assets/js/blocks/cart-checkout/mini-cart/edit.tsx

```diff
@@ -87,6 +94,32 @@ const MiniCartBlock = ( {
 						}
 					/>
 				</PanelBody>
+				{ isSiteEditorAvailable &&
+					isString( themeSlug ) &&
```

Comment:
I don't like this check, but the type of themeSlug is `unknown`. Maybe I can refactor this when [`getSettingWithCoercion` function ](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5184/files#diff-0d3952ed1d1f5ca7da0fbccaaab5bb07976920377492db9ce119d91f80a5b5d3R28)will be merged